x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -565,7 +587,9 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
             <Fragment>
               <EuiFormRow
                 fullWidth
-                isInvalid={maxDistinctValuesError !== undefined}
+                isInvalid={
```

Comment:
I wonder if it might make more sense to show the invalid field warning below the 'Excluded fields' control, as it's most likely to appear when the user selects too many fields to exclude?

Reply:
🤔  Good point, @peteharverson. I'll go ahead and move it 👌